File: Source/Jekov.Nevix/Common/Jekov.Nevix.Common.ViewModels/UserLoginViewModel.cs
﻿namespace Jekov.Nevix.Common.ViewModels
{
    using System.ComponentModel.DataAnnotations;
    using System.Runtime.Serialization;

    [DataContract]
    public class UserLoginViewModel
    {
        [Required]
        [DataMember(Name = "email")]
        public string Email { get; set; }

        [Required]
        [DataMember(Name = "password")]
        public string Password { get; set; }
    }
}

File: Source/Android/Nevix/nevix/src/main/java/com/antonyjekov/nevix/viewmodels/UserLoginViewModel.java
package com.antonyjekov.nevix.viewmodels;

public class UserLoginViewModel {
    private String email;
    private String password;

    public UserLoginViewModel(String email, String pass){
        this.email = email;
        this.password = pass;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}
